src/HashBus.Viewer/packages.config

```diff
@@ -4,5 +4,5 @@
   <package id="GitVersionTask" version="3.6.2" targetFramework="net451" developmentDependency="true" />
   <package id="LiteGuard" version="0.10.0" targetFramework="net451" />
   <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net451" />

```

Comment:
Should these other entries also be changed to `net461`?

Reply:
yeah... I'll recheck this